dashboard/common/db_util.py

```diff
@@ -82,12 +84,12 @@ def get_default_user_course_id(user_id):
     return course_id
 
 
-def get_user_courses_info(user_id):
+def get_user_courses_info(username):
     logger.info(get_user_courses_info.__name__)
     course_list = []
-    course_info={}
+    course_info = []
     with django.db.connection.cursor() as cursor:
-        cursor.execute("SELECT course_id FROM user WHERE sis_name= %s", [user_id])
+        cursor.execute("SELECT course_id FROM user WHERE sis_name= %s", [username])
```

Comment:
so the user_id should have been username prior to the change? Nice catch!

Reply:
Yeah, looks like it was just renaming the value local to this method.